src/Components/Stats/Stats.js

```diff
@@ -1,38 +1,353 @@
+                    <img
+                      src={determineFace(moodStatistics.movementAverageMood2).icon}
+                      className={style.moodIcon}
+                      style={{ backgroundColor: determineFace(moodStatistics.movementAverageMood2).color }}
+                    />
+                  }
+                  <p 
+                    className={style.moodStatisticValue}
+                    data-testid='movementMood2'
+                  >
+                    {moodStatistics.movementAverageMood2.toFixed(2)}
+                  </p>
+                </div>
+              </div>
+            </h4>
+            <h4 className={style.moodWidget}>
+              Meditation/Breathwork
+              <p className={style.moodAverageLabel}>
+                Average Mood
+              </p>
+              <div className={style.moodStatsContainer}>
+                <div className={style.moodStatsSection}>
+                  <p className={style.moodWidgetLabel}>
+                    Before rest interval
+                  </p>
+                  {moodStatistics.meditationAverageMood1 > 0 &&
+                    <img
+                      src={determineFace(moodStatistics.meditationAverageMood1).icon}
+                      className={style.moodIcon}
+                      style={{ backgroundColor: determineFace(moodStatistics.meditationAverageMood1).color }}
+                    />
+                  }
+                  <p 
+                    className={style.moodStatisticValue}
+                    data-testid='meditationMood1'
+                  >
+                    {moodStatistics.meditationAverageMood1.toFixed(2)}
+                  </p>
+                </div>
+                <div className={style.line} />
+                <div className={style.moodStatsSection}>
+                  <p className={style.moodWidgetLabel}>
+                    After rest interval
+                  </p>
+                  {moodStatistics.meditationAverageMood2 > 0 &&
+                    <img
+                      src={determineFace(moodStatistics.meditationAverageMood2).icon}
+                      className={style.moodIcon}
+                      style={{ backgroundColor: determineFace(moodStatistics.meditationAverageMood2).color }}
+                    />
+                  }
+                  <p 
+                    className={style.moodStatisticValue}
+                    data-testid='meditationMood2'
+                  >
+                    {moodStatistics.meditationAverageMood2.toFixed(2)}
+                  </p>
+                </div>
+              </div>
+            </h4>
+          </section>
+        </> :
+        <h2 className={style.newUserMessage}>
+          You don't have any information to display yet! Spend some time using Som Timer and then visit this page again to see your Stats.
+        </h2>
+      }
+    </>
+  )
 }
 
 export default Stats
-
 Stats.propTypes = {
   toggleTimerView: PropTypes.func
 }
```

Comment:
This is all really amazing Jake, excellent work. I love seeing all the logic here that isn't just about rendering pieces of UI. I think in the future, it would be worth it to spend some time breaking this big momma component down into smaller components - I can see each widget section or type of statistic being it's own component. Obviously no pressure to do that now, but I think that could be a good future iteration for us with this feature.